lib/features/manage_account/presentation/profiles/identities/widgets/identities_radio_list_builder.dart

```diff
@@ -10,7 +10,7 @@ import 'package:tmail_ui_user/features/manage_account/presentation/profiles/iden
 import 'package:tmail_ui_user/features/manage_account/presentation/profiles/identities/widgets/identity_list_tile_builder.dart';
 import 'package:tmail_ui_user/features/manage_account/presentation/profiles/identities/widgets/signature_builder.dart';
 
-class IdentitiesRadioListBuilder extends StatelessWidget {
+class IdentitiesRadioListBuilder extends StatefulWidget {
```

Comment:
why we need to move it to `StatefulWidget`?

Reply:
Before this upgrade, `FadingEdgeScrollView.fromScrollView` has `shouldDisposeScrollController` property, and if we set it to true, `FadingEdgeScrollView` will dispose of the `ScrollController` for us. But from version 4.0.0:
![Screenshot 2024-07-10 at 10 40 55](https://github.com/linagora/tmail-flutter/assets/160106668/b9f1bc26-312d-42e3-8b76-ef2b28a97156)
Hence we have to manage the lifecycle of `ScrollController` on our own. That's why I changed the `Stateless` to `Stateful`, to dispose of the `ScrollController`.

Reply:
> Before this upgrade, `FadingEdgeScrollView.fromScrollView` has `shouldDisposeScrollController` property, and if we set it to true, `FadingEdgeScrollView` will dispose of the `ScrollController` for us. But from version 4.0.0: ![Screenshot 2024-07-10 at 10 40 55](https://private-user-images.githubusercontent.com/160106668/347250031-b9f1bc26-312d-42e3-8b76-ef2b28a97156.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA1ODQ1MDAsIm5iZiI6MTcyMDU4NDIwMCwicGF0aCI6Ii8xNjAxMDY2NjgvMzQ3MjUwMDMxLWI5ZjFiYzI2LTMxMmQtNDJlMy04Yjc2LWVmMmIyOGE5NzE1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxMFQwNDAzMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NjQ2NTY1MTIxMmJlYTk5ODNjNzE2OWFiZmFiMGNmODg5YTNlNGZlYTRhMmQyZjM5N2Y0N2ZjNThjYzE5NWEyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.awD3aQcElbs11FWoIZu2QFgpLCeIQKOpDqqgnqUibaA) Hence we have to manage the lifecycle of `ScrollController` on our own. That's why I changed the `Stateless` to `Stateful`, to dispose of the `ScrollController`.

How does it throw exception when we try to use `StatelessWidget`? Please comment errors here for easy tracking.

Reply:
It does not throw any exception when using `StatelessWidget`. Updating this widget to `StatefulWidget` is only for accessing the lifecycle methods of it, so we can dispose the `ScrollController`. If we use `StatelessWidget`, how can we do that?

Reply:
> It does not throw any exception when using `StatelessWidget`. Updating this widget to `StatefulWidget` is only for accessing the lifecycle methods of it, so we can dispose the `ScrollController`. If we use `StatelessWidget`, how can we do that?

Why do we need to remove it? What if `ScrollController` is not removed? We need to clarify why we need it, not because of some library change we follow.

Reply:
>Why do we need to remove it?

Why do we need to remove resources? To avoid memory leak of course

Reply:
> > Why do we need to remove it?
> 
> Why do we need to remove resources? To avoid memory leak of course

So, Please provide evidence of the memory leak here.